modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc

```diff
@@ -107,67 +102,31 @@ To be prepared for the migration, both sides need to match the prerequisites. Pl
 * As a major prerequisite, the ownCloud instance *must* be running on release 10.14 or higher. If this requirement is not met, migration can not be started as the necessary app checks the minimum version.
 
 * Shell access is necessary. +
-`occ` commands need to be started and `rclone` needs to be granted executable permissions.
+`occ` commands need to be started .
 
-* The ownCloud-provided migration components must be installed at the ownCloud instance. They are:
+* The ownCloud-provided migration component must be installed at the ownCloud instance. They are:
```

Comment:
```suggestion
* The ownCloud-provided migration component must be installed at the ownCloud instance. It is:
```

Since there is only one thing now, maybe a list with one item is not needed.